core/src/main/java/io/parsingdata/metal/token/Def.java

```diff
@@ -29,35 +29,27 @@
 import io.parsingdata.metal.data.ParseValue;
 import io.parsingdata.metal.data.Slice;
 import io.parsingdata.metal.encoding.Encoding;
-import io.parsingdata.metal.expression.Expression;
-import io.parsingdata.metal.expression.True;
 import io.parsingdata.metal.expression.value.Value;
 import io.parsingdata.metal.expression.value.ValueExpression;
 
 /**
  * A {@link Token} that specifies a value to parse in the input.
  * <p>
- * A Def consists of a <code>size</code> (a {@link ValueExpression}, in bytes)
- * and a <code>predicate</code> (an {@link Expression}). The
- * <code>predicate</code> may be <code>null</code>.
+ * A Def consists of a <code>size</code> (a {@link ValueExpression}.
  * <p>
- * Parsing will succeed if <code>size</code> evaluates to a single value, if
- * that many bytes are available in the input and if <code>predicate</code>
- * (if present) evaluates to <code>true</code>.
+ * Parsing will succeed if <code>size</code> evaluates to a single value and if
+ * that many bytes are available in the input.
```

Comment:
There are _always_ 0 bytes available right? But I agree it needs to be mentioned explicitly.